cirq/ops/fsim_gate_test.py

```diff
@@ -217,9 +217,20 @@ def test_fsim_unitary():
     )
 
 
-def test_trace_distance():
-    foo = sympy.Symbol('foo')
-    assert cirq.trace_distance_bound(cirq.FSimGate(foo, foo)) == 1
-    assert cirq.trace_distance_bound(cirq.FSimGate(np.pi + 0.01, 0)) == 1
-    assert np.isclose(cirq.trace_distance_bound(cirq.FSimGate(0, np.pi / 2)),
-                      np.sin(np.pi / 4))
```

Comment:
Did this get accidentally deleted?

Reply:
I originally had some edits in fsim_gate that got removed and therefore these additional tests became unnecessary. I only recently noticed that while I was resolving a merge conflict.